### DISCUSSION TITLE
"order by" appeared twice in generated sql which cause side effect - " Unknown column 'Model.createdAt' in 'order clause'"

### DISCUSSION THREAD

**Discussion by theromie** [sequelize/sequelize#15783] [Help & Questions] [answered]
Here is the link: [Issue Link](https://github.com/sequelize/sequelize/issues/15776)

**Comment by WikiRik** [comment, marked as answer]
Please don't make a new discussion but update your original issue. I'm copying the message from the issue below;

---

Hi! We get a lot of issues, which takes a lot of time to process.
For this reason, we have to ask you to fill-in the [issue template](https://raw.githubusercontent.com/sequelize/sequelize/main/.github/ISSUE_TEMPLATE/bug_report.md). The most important part being to provide a [SSCCE](https://github.com/sequelize/sequelize-sscce) that replicates your issue.

As indicated in the issue template, issues that do not follow the template may be closed.
You can edit this issue and ping us, and we will re-review and re-open this issue (there is no need to create a new issue, editing this one is enough).
